ebpf_extensions/pktmonebpfext/sys/PktmonEbpfExt.inf

```diff
@@ -0,0 +1,98 @@
+;
+;  Copyright (c) Microsoft Corporation
+;  SPDX-License-Identifier: MIT
+;
+
+;
+; PktmonEbpfExt.inf
+;
+
+[Version]
+Signature="$WINDOWS NT$"
+Class=EbpfExtension
+ClassGuid={b23b88e3-60fb-4950-b71e-2f70e591dff6}
+Provider=%ManufacturerName%
+CatalogFile=PktmonEbpfExt.cat
+DriverVer= 1/11/2021,1.0.0.0
+PnpLockDown=1
```

Comment:
I don't know what `DriverVer` influences, but should it have a date closer to now?

Reply:
Please consider this branch is still under heavy development, not ready for reviews yet.